src/components/screens/home/balance-card.tsx

```diff
@@ -19,6 +21,10 @@ const BalanceCard = () => {
   const setError = useSetAtom(errorAtom);
   const [loading, setLoading] = useState(false);
   const [personTotalTime, setPersonTotalTime] = useState<PersonTotalTime>();
+  const currentDate = DateTime.now();
+  const beforeDate = currentDate.minus({ days: 1 }).set({ hour: 23, minute: 59 });
+  const language = useAtomValue(languageAtom);
+  const formattedBeforeDate = beforeDate.setLocale(language).toFormat("dd LLL yyyy");
```

Comment:
Move this logic to `App.tsx`. Therefore all luxon created DateTimes will be formatted to the same global locale and it avoids repetition.

for example
```ts
...
import * as luxon from "luxon";
import { useLayoutEffect } from "react";

const App = () => {
  const currentLocale = useAtomValue(languageAtom);
  useLayoutEffect(() => {
    luxon.Settings.defaultLocale = currentLocale;
  });
...
};
```

Reply:
Good idea. I implemented this but with useMemo so that the state change wouldn't lag one render behind.